python/foglamp/plugins/south/envirophat/envirophat.py

```diff
@@ -222,8 +222,9 @@ def plugin_reconfigure(handle, new_config):
 
     # Plugin should re-initialize and restart if key configuration is changed
     if 'pollInterval' in diff:
-        new_handle = copy.deepcopy(new_config)
-        new_handle['restart'] = 'no'
+        plugin_shutdown(handle)
+        new_handle = plugin_init(new_config)
+        new_handle['restart'] = 'yes'
```

Comment:
Better if we add `_LOGGER.info(“Restarting Enviro pHAT plugin due to change in configuration key [{}]“.format(‘, ’.join(diff)))`